### PR TITLE
fix: changing the bootstrapping suggestion to the command

### LINKED ISSUE #35526: Initial keycloak bootstrap suggestion is not correct.
The suggestion to use the env variables in the index template:

"To create the administrative user open <a href="${localAdminUrl}">${localAdminUrl}</a>, or set the environment variables <code>KC_BOOTSTRAP_ADMIN_USERNAME</code> and <code>KC_BOOTSTRAP_ADMIN_PASSWORD</code> when starting the server."

Is not correct - it is too late to use the environment variables because they are only used when the master realm is created, which would have already happened.

_Originally posted by @shawkins in https://github.com/keycloak/keycloak/discussions/35477#discussioncomment-11439894_

### PR DESCRIPTION
**Pull request by shawkins**
closes: #35526

cc @jonkoops @vmuzikar the admin message from the welcome resource doesn't seem to be used by us, but I'm not sure if it could be used by something custom and therefore needs to stay for now.

### CODE REVIEW

File: services/src/main/java/org/keycloak/services/resources/WelcomeResource.java [hunk 1]
```diff
@@ -255,7 +255,7 @@ private Theme getTheme() {
     }
 
     protected String getAdminCreationMessage() {
```
Comment (jonkoops) [inline services/src/main/java/org/keycloak/services/resources/WelcomeResource.java:257]: Does this actually get used anywhere? Perhaps we can get rid of this code.

**Review by jonkoops — APPROVED** [review]
LGTM

File: services/src/main/java/org/keycloak/services/resources/WelcomeResource.java (hunk 1 above)
Comment (vmuzikar) [reply services/src/main/java/org/keycloak/services/resources/WelcomeResource.java:257]: The [`adminUserCreationMessage` property](https://github.com/keycloak/keycloak/blob/0eb0281bf222d7781dbf2bdc084a4f73f06c8bc7/services/src/main/java/org/keycloak/services/resources/WelcomeResource.java#L220) doesn't seem to be used, you're right. I don't think we need to wait to 27 before removing it.

**Review by vmuzikar — APPROVED** [review]

File: services/src/main/java/org/keycloak/services/resources/WelcomeResource.java (hunk 1 above)
Comment (vmuzikar) [reply services/src/main/java/org/keycloak/services/resources/WelcomeResource.java:257]: Actually, we can't remove as custom themes might be using it. So we need to wait to 27...

Comment (shawkins) [reply services/src/main/java/org/keycloak/services/resources/WelcomeResource.java:257]: Should I capture this as a follow up issue then?

Comment (jonkoops) [reply services/src/main/java/org/keycloak/services/resources/WelcomeResource.java:257]: Ok, then let's log a task to remove it in the next major

**Pull request merged**